<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainerWindow</class>
 <widget class="QMainWindow" name="TrainerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_5" rowstretch="1,8">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="styleSheet">
         <string notr="true">text-align:top center;</string>
        </property>
        <property name="text">
         <string>Старт</string>
        </property>
        <property name="icon">
         <iconset resource="../../res/resources.qrc">
          <normaloff>:/images/play_button.svg</normaloff>:/images/play_button.svg</iconset>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="styleSheet">
         <string notr="true">text-align:top center;</string>
        </property>
        <property name="text">
         <string>Стоп</string>
        </property>
        <property name="icon">
         <iconset resource="../../res/resources.qrc">
          <normaloff>:/images/stop_button.svg</normaloff>:/images/stop_button.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="breathe_trainer::TimeMeter" name="timeMeter" native="true"/>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Общее время:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="phase">
        <property name="text">
         <string>Phase</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="phaseTime">
        <property name="text">
         <string>Phase Time</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Время:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Фаза:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="totalTime">
        <property name="text">
         <string>Total time</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>breathe_trainer::TimeMeter</class>
   <extends>QWidget</extends>
   <header location="global">TrainerWindow/TimeMeter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
